__tests__/app.test.js

```diff
@@ -46,3 +34,37 @@ describe('/api', () => {
         });
     });
 });
+
+describe('/api/articles/:article_id', () => {
+    test('GET:200 sends a single article to the client', () => {
+        return request(app)
+        .get('/api/articles/1')
+        .expect(200)
+        .then((response) => {
+        expect(response.body.article.article_id).toBe(1);
+        expect(response.body.article.author).toBe('butter_bridge');
+        expect(response.body.article.title).toBe('Living in the shadow of a great man');
+        expect(response.body.article.body).toBe('I find this existence challenging');
+        expect(response.body.article.topic).toBe('mitch');
+        expect(response.body.article.created_at).toBe('2020-07-09T20:11:00.000Z');
+        expect(response.body.article.votes).toBe(100);
+        expect(response.body.article.article_img_url).toBe("https://images.pexels.com/photos/158651/news-newsletter-newspaper-information-158651.jpeg?w=700&h=700");
```

Comment:
This works and is fine for now.
For future tests look at jest matchers expect.objectContaining or .toMatchObject, they allow for more robust and flexible testing.